refract-spec.md

```diff
@@ -498,32 +480,34 @@ The following shows a link with the relation of `foo` and the URL of `/bar`.
 }
 ```
 
-## Namespacing
+## Profiles
 
-Namespaces provide a way to include element definitions and constraints into another document. The following rules and constraints apply to namespaces.
+The primary means by which users can provide semantic definitions and other meta information is through a profile. A profile MAY provide semantic information about an element and its data, it MAY provide specific instructions about elements such as how inheritance should work or how elements should be processed, and it MAY be used to modify understanding of existing elements in other profiles. The usage is a profile is not limited to these descriptions, and SHOULD be left up to the profile author to define its use.
 
-1. If there is a conflict, the namespaced element defined or referenced last MUST take precedence
-1. Types defined within the local namespace MUST take precedence over referenced namespaces
-1. Prefixed namespaces are only accessible through using a prefix
-1. Namespaces apply to the element in which they are introduced and all child elements
-1. Namespaces MUST not include the elements nor the data in the elements referenced, but rather include the constraints and conditions around the types defined within that namespace
+To point to a profile, you MAY use the [profile link relation](https://www.ietf.org/rfc/rfc6906.txt) as a meta link in your root element or in any other element. Profile links may also be found outside of the document itself in places like the [HTTP Link Header](http://www.w3.org/wiki/LinkHeader).
 
-### Example Using Both String and Object Namespace Elements
+Below is an example of how a profile link is used as a meta link in Refract.
 
-This example shows a namespace being included by way of a string, and a prefixed namespace.
-
-```json
+```js
 {
   "element": "foo",
   "meta": {
-    "namespaces": [
-      "http://example.com/namespace1",
-      { "prefix": "ns2", "href": "http://example.com/namespace2" }
+    "links": [
+      {
+        "element": "link",
+        "attributes": {
+          "relation": "profile",
+          "href": "http://example.com/profiles/foo"
```

Comment:
You may want to read through [RFC 6906](https://www.ietf.org/rfc/rfc6906.txt). It was an interesting way to think about things. Check this out.

> The target URI does not necessarily have to identify a
> dereferencable resource (or even use a dereferencable URI scheme).
> Clients can treat the occurrence of a specific URI in the same way as
> an XML namespace URI and invoke specific behavior based on the
> assumption that a specific profile target URI signals that a resource
> representation follows a specific profile.  Note that, at the same
> time, it is possible for profile target URIs to use dereferencable
> URIs and to use a representation (which is outside the scope of this
> specification) that represents the information about the profile in a
> human- or machine-readable way.

The URI actually could return a 404 and be valid. So you can use it as:

1. Machine-readable doc
1. Human-readable doc
1. An identifier of some sort
1. Meta data (like with RDF)

So to "does it contain plain text" or "machine readable," the answer is yes :) And it could be used like the XHTML URL`http://www.w3.org/1999/xhtml` is used. If that URL is there, it means it's XHTML... doesn't matter what is found when you resolve that.

Hope that makes sense.